transport/transport-runtime/src/main/java/com/google/android/datatransport/runtime/logging/Logging.java

```diff
@@ -18,22 +18,24 @@
 import android.util.Log;
 
 public final class Logging {
+  private static final String prefix = "TRuntime.";
   private Logging() {}
 
   private static String getTag(String tag) {
-    String prefix = "TRuntime.";
-    return prefix + concatTag(prefix, tag);
+    if(android.os.Build.VERSION.SDK_INT < Build.VERSION_CODES.N)
+      return concatTag(prefix, tag);
+
+    return prefix + tag;
   }
 
   private static String concatTag(String prefix, String tag){
+    String concatText = prefix + tag;
     int loggableLimit = 23;
```

Comment:
This also looks like a constant. Use a name that reflects that the limit is for N and below, maybe LOGGING_TAG_MAX_SIZE_SDK_N (I'm not that good with names).

Reply:
Got it. I've used the similar format from other constants in other files, where the `MAX` is the starting keyword for the constants.